### DISCUSSION TITLE
Should notifications be turned on by default for logged in users?

### DISCUSSION THREAD

**Discussion by lukevella** [lukevella/rallly#635] [Polls]
When you create a poll a poll as a registered and logged in user, we currently turn notifications on by default. I'm considering changing this behaviour so that users would need to opt-in to notifications instead. This prevents a user getting spammed with emails unexpectedly. Curious to know what people think about this.

**Comment by PLP-GTR** [comment]
I don't know if there are power-users out there which get "spammed" by mails. But those might prefer a option to set the default value in their profile.
